cmd/ctr/commands/run/run.go

```diff
@@ -105,7 +105,7 @@ var Command = cli.Command{
 		},
 		cli.BoolFlag{
 			Name:  "detach,d",
-			Usage: "detach from the task after it has started execution",
+			Usage: "detach from the task after it has started execution, cannot be specified with --rm together",
```

Comment:
same comment as above, the added text could be simplified: `, cannot be used with --rm`

Reply:
updated